### ISSUE TITLE
Styleguide page

### ISSUE THREAD

**Issue by cycomachead** [berkeley-cdss/berkeley-class-site#39]
We should have 1 page in the template which tries to show all:

- labels
- foreground/background color contrast combos
- important UI elements
